cyences_app_for_splunk/default/data/ui/views/cs_network_reports.xml

```diff
@@ -1,5 +1,6 @@
 <form theme="dark" version="1.1">
   <label>Network Reports</label>
+  <description>Note: For the larger time range period map won't render all the events as the commands used in the search query (like. mvexpand, streamstat) has some limitation.</description>
```

Comment:
Note: For the larger time range map might not render all the events as the commands used in the search query (like mvexpand, streamstat) has some limitation set.